### DISCUSSION TITLE
FastZip vs Zip performance

### DISCUSSION THREAD

**Discussion by Shloyem** [icsharpcode/SharpZipLib#557] [Q&A]
Hi,

I am trying to choose between using FastZip and Zip, and was wondering about FastZip:

1. Is FastZip simply a wrapper above regular Zip library? That is meant to ease and shorten the use.
2. If so, should I expect the same performance(required time, file size) using either one of them? Given the same compression level, encryption key length, etc.

Thanks in advance

**Comment by piksel** [comment]
1. Yeah, exactly. It's for the "just do it, I don't care" use case.
2. Pretty much. If you do it the "manual" way you can decide how to iterate the contents and perhaps only extract certain entries etc., but the raw decompression and decryption should be equivalent.
